src/tools/bin/nextstep_from_period.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017-2018 Axel Huebl
```

Comment:
@wmles Please add your name to the author list, or was this code entirely written by @ax3l ?